src/components/typography.js

```diff
@@ -8,7 +8,7 @@ Text.defaultProps = {
   fontSize: 'text',
   textAlign: 'left',
   padding: 0,
-  margin: 'textMargin'
+  margin: 0
```

Comment:
The idea here is to use an index value of the space scale instead of fixed values as defined by the default theme. That makes it nearly impossible for the user to easily override them.